src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -42,6 +42,132 @@ class OrchestratorCli(orchestrator.OrchestratorClientMixin, MgrModule):
         {'name': 'orchestrator'}
     ]
 
+    def __init__(self, *args, **kwargs):
```

Comment:
You mean the CLI implementation or the interface? 

For the CLI, I'd like to make sure we only have one instance of this Implementation. For the interface, that would be an idea, but I was lacking the need for making it more complicated.

Reply:
In this case I mean the implementation. Instead of having a big class as a container for multiple somehow unrelated behaviours, perhaps moving those to smaller specific classes. That'd be the idea.

Reply:
hmm. Most of the CLI implementation is quite boring: it just forwards the command to the external orchestrator. Would you be ok with refactoring this into a separate mixin, as soon as the Interface implements other methods that need to store some other state?

Reply:
To sum up my proposal:
- Using an `Interface` in the Orchestrator (e.g.: `CanBlink`), so any orchestrator implementation needs to explicitly state that it inherits (and hence fully implements) from the `CanBlink` inteface (that way you could use Python `abc`).
- Using a `Mixin` in the CLI to break the big class into functionally related implementations (e.g.: `BlinkMixin`). 

However, don't worry too much about it, @sebastian-philipp, I already gave my ok to this :wink: